src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -388,7 +388,7 @@ public override DownloadClientInfo GetStatus()
                 }
             }
 
-            var minimumRetention = 60 * 24 * 14;
+            var minimumRetention = GetMinimumRetention();
```

Comment:
Instead of extracting the minimum value, extract the logic for calculating `RemovesCompletedDownloads` that way it can be re-used in `TestConnection` without duplicating it.

Reply:
Fixed